src/components/TextInput/TextInput.stories.tsx

```diff
@@ -85,10 +92,64 @@ const BetterExample: FC<StoryProps> = (props: StoryProps) => {
   return <TextInput {...mutatedProps} value={value} ref={ref} onChange={onChange} />;
 };
 
+const FormExample: FC<FormProps> = (props: FormProps) => {
+  const { displayName = false, emailAddress = false, password = false } = props;
+
+  return (
+    <>
+      <form>
```

Comment:
Please add the following:
```suggestion
    <form
      onSubmit={(e) => {
        e.preventDefault();
        location.reload();
      }}
    >
```
this will prevent to reload the whole storyboard, but it will reload the `iframe` of the story which triggers the browser password manager to store the email / password